.kitchen.yml

```diff
@@ -206,6 +206,31 @@ suites:
       - recipe[hadoop::zookeeper]
       - recipe[hadoop::zookeeper_server]
     attributes: { hadoop: { distribution: 'bigtop', distribution_version: '0.8.0' } }
+  - name: bigtop100
+    excludes: debian-6.0.10, ubuntu-14.04
```

Comment:
A quick test on Ubuntu 12 overriding the codename will install successfully and I could start the hdfs namenode and zookeeper.

I don't have time to test everything thoroughly but if that's satisfactory, I can commit the codename override hack.